reactnative/ReactNativeTemplateApp/package.json

```diff
@@ -6,6 +6,6 @@
     "start": "node_modules/react-native/packager/packager.sh --root js/"
   },
   "dependencies": {
-    "react-native": "^0.20.0"
+    "react-native": "^0.27.2"
```

Comment:
We'll have to keep that version number lined up with the one we npm install in install.sh, otherwise the SDK repo and generated app will work against different versions of react native.